backend/routes/itemRoutes.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const router = express.Router();
+const ITEM = ('../item-schema.js');
+
+router.get(ITEMid, getITEM);
```

Comment:
these routes need to be updated to something like '/item'
that will be sufficient for now.